script/sdistsign.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+set -e
+
+KEY_ID=2E390FA0
+KEY_FILE=${SDIST_KEY_FILE:-signing.key}
+
+version=$(python setup.py --version)
+dist_dir=${STREAMLINK_DIST_DIR:-dist}
+temp_keyring=$(mktemp -d) && trap "rm -rf ${temp_keyring}" EXIT || exit 255
+
+if [[ -n "${TRAVIS}" ]]; then
+      openssl aes-256-cbc -K ${encrypted_eeb8b970d3a3_key} -iv ${encrypted_eeb8b970d3a3_iv} -in signing.key.enc -out "${SDIST_KEY_FILE}" -d
+fi
+
+echo "build: Installing twine and wheel" >&2
+pip -q install twine wheel
```

Comment:
`wheel` can be removed, it won't work with **python 2.7**

```
$ pip list
Package    Version
---------- -------
pip        10.0.1 
setuptools 39.2.0 
wheel      0.31.1 
```
```
$ pip install streamlink
Collecting streamlink
  Using cached .../streamlink-0.13.0-py2.py3-none-any.whl
  ...
Installing collected packages: iso-639, iso3166, six, websocket-client, chardet, certifi, urllib3, idna, requests, pycryptodome, PySocks, isodate, streamlink
Successfully installed PySocks-1.6.8 certifi-2018.4.16 chardet-3.0.4 idna-2.6 iso-639-0.4.5 iso3166-0.8 isodate-0.6.0 pycryptodome-3.6.1 requests-2.18.4 six-1.11.0 streamlink-0.13.0 urllib3-1.22 websocket-client-0.48.0
```

```
$ streamlink 
...
ImportError: No module named backports.shutil_which
```

```
$ pip list
Package          Version  
---------------- ---------
certifi          2018.4.16
chardet          3.0.4    
idna             2.6      
iso-639          0.4.5    
iso3166          0.8      
isodate          0.6.0    
pip              10.0.1   
pycryptodome     3.6.1    
PySocks          1.6.8    
requests         2.18.4   
setuptools       39.2.0   
six              1.11.0   
streamlink       0.13.0   
urllib3          1.22     
websocket-client 0.48.0   
wheel            0.31.1 
```

Reply:
Can add a fix for this :)

Reply:
@back-to would you be able to test out building a whl with #1758 and seeing that installs for you? I assume this is an issue for Enigma2 boxes?